src/contracts/sc-options.js

```diff
@@ -0,0 +1,75 @@
+import { keccak256 } from "@ethersproject/keccak256";
```

Comment:
sc in sc-options stutters on some extent because we are already in the contracts folder. I think you can rename this to options

Reply:
good point